CONTRIBUTING.md

```diff
@@ -1,11 +1,40 @@
 # How to Contribute
 
-TODO: docs on how to run the nextjs project and maybe even the fork/clone/pr flow
+1. Fork the repository
+2. Clone the repository to your local machine
+3. Create a new branch 
+```
+ git checkout -b <branch-name>
+```
 
-## Environment Variables
+4. Make your changes
+5. Commit and push your changes
+```
+ git add .
+ git commit -m "commit message"
+ git push origin <branch-name>
+```
+6. Create a pull request
+7. Wait for the pull request to be reviewed and merged
+
+
+# How to Setup Environment Variables
 
 Rename the file `.env.example` into `.env.local` and fill in the values.
```

Comment:
This was already wrong but since you're on this file would you mind changing it?
It should be "Duplicate and rename" otherwise the example is gone (as you can see in this PR it's marked as deleted)

Reply:
yeah sure, it was my bad